src/AppInstallerCommonCore/MsixInfo.cpp

```diff
@@ -335,6 +556,18 @@ namespace AppInstaller::Msix
         THROW_IF_FAILED(signatureStream->Stat(&stat, STATFLAG_NONAME));
         THROW_HR_IF(E_UNEXPECTED, stat.cbSize.HighPart != 0); // Signature size should be small
         signatureSize = stat.cbSize.LowPart;
+        THROW_HR_IF(E_UNEXPECTED, signatureSize <= P7xFileIdSize);
+
+        if (getRawSignature)

```

Comment:
`getRawSignature` being true, but the code then modifying the signature, doesn't make sense to me.  Maybe this was supposed to be `if (!getRawSignature)`?  Or should the name be changed to `skipFileID`?